plugin.xml

```diff
@@ -37,7 +37,7 @@
         <source-file src="src/android/io/sqlc/SQLitePlugin.java" target-dir="src/io/sqlc"/>
         <source-file src="src/android/io/sqlc/SQLiteAndroidDatabase.java" target-dir="src/io/sqlc"/>
 
-        <lib-file src="src/android/libs/android-database-sqlcipher-4.5.4-custom.aar" />
```

Comment:
If the aar is not being used anymore, shouldn't it be deleted from the repo?

Reply:
Correct yes, thanks for pointing it out!

Done in https://github.com/OutSystems/Cordova-sqlcipher-adapter/pull/14/commits/0739cbef5ff4d9d30e87c0a69e0f7ce6c83476b6